mig-scheduler/collector.go

```diff
@@ -134,6 +134,12 @@ func loadReturnedCommands(ctx Context) (err error) {
 			continue
 		}
 		filename := ctx.Directories.Command.Returned + "/" + DirEntry.Name()
+
+		if strings.Index(filename, ".fail") == len(filename)-5 {
```

Comment:
Why not just use `strings.HasSuffix()` ? 
https://golang.org/pkg/strings/#HasSuffix